ui/app/components/ui/list-item/list-item.component.js

```diff
@@ -57,6 +61,7 @@ export default function ListItem ({
 
 ListItem.propTypes = {
   title: PropTypes.string.isRequired,
```

Comment:
What if, instead of having a new prop, we upgrade title to be ```PropTypes.oneOf([PropTypes.string, PropTypes.node])```

then we could do something like
```js
{React.isValidElement(title) ? title : <h2>title</h2>}
```

then you can move the styles and creation of the subtitle to the asset-list-item instead of the list-item.

Reply:
```
title={(
 <>
   <h2 className="asset-list-item__title">{value}</h2>
   <h2 className="asset-list-item__subTitle">{symbol}</h2>
</>
)}
```

Reply:
you could also use more descriptive names for the classes then and avoid the double usage of 'subtitle' surrounding list-item.

Reply:
82fc160